website/package.json

```diff
@@ -5,20 +5,20 @@
   "scripts": {
     "start": "docusaurus-start",
     "build": "docusaurus-build",
-    "gh-pages": "docusaurus-publish",
+    "gh-pages": "echo 'replaced by Netlify deployment!'",
     "examples": "docusaurus-examples",
-    "crowdin-upload": "crowdin --config ../crowdin.yaml upload sources --auto-update -b master",
-    "crowdin-download": "crowdin --config ../crowdin.yaml download -b master",
-    "publish-gh-pages": "docusaurus-publish",
+    "crowdin-upload": "java -jar crowdin-cli-v2.jar --config ../crowdin.yaml upload sources --auto-update -b master",
```

Comment:
what's wrong with `crowdin-cli`? adding a 11 mb jar file feels a bit icky

Reply:
It's not possible to install deb files in Netlify, and the npm package is too new (cli v3 while we need v2).
The only way is to use the jar directly (which is what both npm+deb+rpm use anyway).

If you don't want to add this jar to git, the most I can do is to self-host it somewhere else and download it with curl/wget, so it's just a bit more likely to break the deployment if the jar disappears and becomes unavailable for downloading

Reply:
>(cli v3 while we need v2)

Any chance to fix it so we're on latest? Up until that point having the jar in the repo seems fine 👍

Reply:
No, for some technical reasons (Crowdin behavior), we actually need the legacy v2 cli on purpose, like the current production cli version, as we download the translations from the same Crowdin project.

Otherwise I'd have to update production too, which would be risky and quite useless for the time it will stay in life :)